docs/source/changelog.md

```diff
@@ -1,7 +1,35 @@
-# ipywidgets changelog
+# ipywidgets Changelog
 
 A summary of changes in ipywidgets. For more detailed information, see the issues and pull requests for the appropriate milestone on [GitHub](https://github.com/jupyter-widgets/ipywidgets).
 
+## 7.6
+
+To see the full list of pull requests and issues, see the [7.6.0 milestone](https://github.com/jupyter-widgets/ipywidgets/milestone/31?closed=1) on GitHub.
+
+The main change in this release is that installing `ipywidgets` 7.6.0 will now automatically enable ipywidgets support in JupyterLab 3.0—a user has no extra JupyterLab installation step and no rebuild of JupyterLab, nor do they need Node.js installed. Simply install the python ipywidgets package with pip (`pip install ipywidgets==7.6.0`) or conda/mamba (`conda install -c conda-forge ipywidgets=7.6.0`) and ipywidgets will automatically work in classic Jupyter Notebook and in JupyterLab 3.0.
```

Comment:
Should we change the doc string here to account for patch versions? Changing to 7.6.3 in docs would be good but if a .4 patch is ever needed it can be hard to remember to update these so maybe a `<7.7` would be best?

Reply:
I will handle this in an upcoming doc change after this PR and #3086 land. This way it's a bit easier to align content changes across both branches vs. just structural changes.